tests/client/amo/sagas/testIndex.js

```diff
@@ -0,0 +1,15 @@
+import { fork } from 'redux-saga/effects';
+
+import rootSagas from 'amo/sagas';
+import categoriesSaga from 'amo/sagas/categories';
+
+
+describe('amo rootSagas', () => {
+  it('should get Api from state then make API request to categories', () => {
+    const sagaGenerator = rootSagas();
+
+    assert.deepEqual(sagaGenerator.next().value, [
+      fork(categoriesSaga),
```

Comment:
We do essentially the same thing for reducers right now... though we might be able to use the redux saga testing utils to make it better. I forgot to use them in this test. I'll see about that and failing that do your thing, yeah.